ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master-presubmits.yaml

```diff
@@ -289,23 +289,23 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-azure,?($|\s.*)
+    trigger: (?m)^/test( | .* )e2e-azure-operator,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
These files are generated by `make jobs`, so I trust that this change to `always_run` is required.  I did not make any changes that should have had any impact on this.

Reply:
yeah, if you look down you can see that this section is now for the job named:

pull-ci-openshift-cluster-ingress-operator-master-e2e-azure-ovn and not
pull-ci-openshift-cluster-ingress-operator-master-e2e-azure-operator

I think make jobs is shuffling around the jobs in this file some way or another (alphabetically) and it's retaining the
proper values of "always_run" so this diff ends up looking like this.